scala/lambdas/ingest-mapper/src/test/scala/uk/gov/nationalarchives/ingestmapper/MetadataServiceTest.scala

```diff
@@ -92,22 +92,33 @@ class MetadataServiceTest extends AnyFlatSpec with MockitoSugar with TableDriven
             }
 
           val batchId = "batchId"
-          val folderId = UUID.randomUUID()
-          val assetId = UUID.randomUUID()
-          val fileIdOne = UUID.randomUUID()
-          val fileIdTwo = UUID.randomUUID()
+          val folderIdOne = UUID.randomUUID()
+          val assetIdOne = UUID.randomUUID()
+          val fileIdOneOne = UUID.randomUUID()
+          val fileIdOneTwo = UUID.randomUUID()
```

Comment:
😄 , couldn't we call these `fileIdOne` & `metadataFileOne` or `fileId1` & `metadataFile1`. Same question for the other pair of file ids

Reply:
:smile: I was getting fed up at this point. `fileIdOne` & `metadataFileOne` are good.